TABTestKit/Classes/TABTestCase.swift

```diff
@@ -81,6 +81,11 @@ open class TABTestCase: XCTestCase, DefaultContexts {
 		// You can find this attachment in the .xcresult bundle (usually Derived Data).
 		let attachment = createFailureAttachment(description: description, filePath: filePath, lineNumber: lineNumber)
 		add(attachment)
+
+        if App.shared.screenshotOption.contains(.onFailure) {
```

Comment:
I'm pretty sure XCUI already adds screenshots when a failure happens anyway?

Reply:
> Hello! Thanks for taking the time to do this :) it would be super if you could create issues to discuss changes like this before working on changes because if we decide not to incorporate this then you will have wasted your time

Could you like these two 
https://github.com/theappbusiness/TABTestKit/issues/143
https://github.com/theappbusiness/TABTestKit/issues/144

Reply:
Have to check that, will come back to that